docker-compose.yml

```diff
@@ -32,6 +32,14 @@ services:
       - "8000:80"
     depends_on:
       - app
-
+  queue:
+    image: rabbitmq:alpine
+    hostname: cloudiqueue
```

Comment:
is there a reason this is necessary? Do we not just want to refer to this as `queue`?

Reply:
If we don't specify it gets randomly generated by Docker. RabbitMQ persists data by node/host, creating a directory based on that relationship. If we specify it, it will always load and store data from `rabbit@cloudiqueue`. We won't actually persist data in a reusable way if we don't specify it.